packages/core/dialect/saphana/index.ts

```diff
@@ -0,0 +1,199 @@
+
+import {
+  ConnectionDialect
+} from '@sqltools/core/interface';
+import * as Utils from '@sqltools/core/utils';
+import queries from './queries';
+import GenericDialect from '@sqltools/core/dialect/generic';
+import { DatabaseInterface } from '@sqltools/core/plugin-api';
+// Cannot simply use hanaClient from '@sap/hana-client', because webpack is moving it away from its native libs.
```

Comment:
Take a look at SQLite and Oracle implementation. I believe that this case would be better if handled by the dependencyManager like SQLite and Oracle does

Reply:
done